### DISCUSSION TITLE
I'm under strange situation - email and the website shows different result of the verification process

### DISCUSSION THREAD

**Discussion by Fyphen1223** [community/community#89915] [GitHub Education]
### Hi everyone,

Hi all,

I'm a student, and I've been trying to get students' benefits lately.

This is 4 time I do verification process, and I got an e-mail that says I'm rejected.
![image](https://github.com/community/community/assets/89511960/2613a6e8-7f6d-4549-8b8f-17cad1096b40)

But when I visit benefits page, I got this.
![image](https://github.com/community/community/assets/89511960/b5632245-fc96-4644-92d5-e1b1418e5115)

Which is correct?

**Comment by fury-05** [comment]
Well this is odd, Try visiting this link https://education.github.com/pack/offers this are benefits you get after your student verification if you are able to access and use them then you have verified indeed.

Also on same link when you click on profile  drop down on top right you can refer the below image

![image](https://github.com/community/community/assets/103818328/79b8ac07-23da-48c3-84ca-07ea93a49009)

If you see Reverify your academic affiliation that means you have verified status and can use students benefits.

**Reply by Fyphen1223** [reply]
Thanks for reply! Currently I cannot see such screen, but I think it because there's delay of the arrival of the offer and email(status). I'll wait a few days and see what there will be.

**Reply by fury-05** [reply]
There is conflict in system, I think you should email support to resolve it.

**Reply by Fyphen1223** [reply]
Oh, I see! Thanks for lettming me know :)

**Comment by aadltya** [comment]
Mark this as answer

**Reply by Fyphen1223** [reply]
what

**Reply by aadltya** [reply]
nthing

**Reply by Fyphen1223** [reply]
Ok.
